styles/google-gemini/catppuccin.user.css

```diff
@@ -178,7 +178,21 @@
     --bard-color-tunable-selection-textarea-background: @base;
     --bard-color-tunable-selection-shimmer-non-advanced: @accent-color;
     --bard-color-action-tooltip-background: darken(@accent-color, 20%);
-
+    --bard-color-adv-marketing-check: @text;
+    --gem-sys-color--surface-container: @mantle;
+    --gem-sys-color--on-surface: @text;
+    --gem-sys-color--outline-variant: @subtext0;
+    --gem-sys-color--surface-container-low: @mantle;
```

Comment:
mini code block
![image](https://github.com/user-attachments/assets/5505b084-197f-401a-ad36-9e226278fc8f)

```suggestion
    --gem-sys-color--outline-variant: @subtext0;
    --gem-sys-color--surface-container-high: @surface0;
    --gem-sys-color--surface-container-low: @mantle;
```